src/core_write.cpp

```diff
@@ -144,7 +144,7 @@ void ScriptToUniv(const CScript& script, UniValue& out, bool include_address)
     out.pushKV("type", GetTxnOutputType(type));
 
     CTxDestination address;
-    if (include_address && ExtractDestination(script, address)) {
+    if (include_address && ExtractDestination(script, address) && type != TX_PUBKEY) {
```

Comment:
This code branch is never exercised in the tests. Would be nice to add one or at the very least some release notes.

Reply:
`ScriptToUniv` and `ScriptPubKeyToUniv` are almost similar. The only real difference is that one has `reqSigs` and `addresses`, but the other only has `address`.

I think the best way to fix this issue is just modify `ScriptPubKeyToUniv` to stop returning an array (it makes no sense to show the individual P2PKH of multisig pubkeys) and returning `address` instead, and then removing `ScriptToUniv` completely.